chromium_src/chrome/browser/ui/test/DEPS

```diff
@@ -0,0 +1,3 @@
+include_rules = [
+  "+third_party/googletest/src/googletest/include/gtest/gtest.h",
```

Comment:
`#include "testing/gtest/include/gtest/gtest.h"` is allowed everywhere, so this can be removed.

Reply:
Fixed in d796000bfc0b57e79de47722732e548bd911f558.